### ISSUE TITLE
How to give navigation using Remove Control

### ISSUE THREAD

**Issue by langaliamayank** [korre/android-tv-epg#12]
Hello i need help. how to set navigation in EPG view ?
Is there any way please suggest me how to achieve this ?

**Comment by devnullpointer** [comment]
Yes, is there a way to make EPGEvent focusable for d-pad functionality?
